UKHomeOffice/cerberus-service
# COP-10851 - Adds filtering by rule matches

## Description
This PR adds a multi select filtering option for filtering tasks by ruleIds.

## To Test
- Pull & run against dev
- In the Filters section, click within the `Rule matches` autocomplete and search for rules to apply. This also allows you to add multiple filters.
- Click on apply, tasks with matching ruleIds should be displayed on the task list. The task counts should also reflect the returned result.
